aspnetcore/mvc/controllers/dependency-injection/sample/ControllerDI/Views/Home/Privacy.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@{
+    ViewData["Title"] = "Privacy Policy";
+}
```

Comment:
Super-nit: As with Index.cshtml (above) and Error.cshtml (below), place a line between the Razor block and the `<h1>`.